tools/clang/lib/AST/HlslTypes.cpp

```diff
@@ -949,6 +949,26 @@ QualType GetHLSLResourceTemplateParamType(QualType type) {
 QualType GetHLSLInputPatchElementType(QualType type) {
   return GetHLSLResourceTemplateParamType(type);
 }
+
+CXXRecordDecl *GetHLSLNodeRecordDecl(clang::QualType type) {
```

Comment:
It looks like both PRs I commented about adding this now have two independent implementations of the same function.  See: https://github.com/microsoft/DirectXShaderCompiler/pull/5992/files#diff-bb5c401f39f10a7cc1f6d80ee2eecf7aa94e6d7d6ce9508a2034793e34b74067R90

These changes should sync on the method used.

Reply:
With the changes I suggested there, I think we can go with the version in the other PR.